packages/storybook/.storybook/style.scss

```diff
@@ -61,3 +61,7 @@ a.sbdocs-a:hover {
 button {
   padding: 10px 20px !important;
 }
+
+.docblock-argstable a {
+  font-size: 14px;
```

Comment:
A small Storybook style update that makes the arg table links the same size as the text. Otherwise it looks weird like this:

<img width="371" alt="Screenshot 2025-02-04 at 4 51 16 PM" src="https://github.com/user-attachments/assets/2c575563-6519-4fc2-a6d3-6c49add9dd67" />

Reply:
I just want to add a list of options for the `submit` prop here so that it's clearer what it does (none of this behavior should have changed as part of this ticket):

- `prevent` will trigger the onsubmit callback on the form, but won't submit the form
- `skip` will submit the form but not trigger the onsubmit callback; 
- All other values will trigger the onsubmit and onclick callbacks, then submit the form; in that order.

Reply:
Just an FYI for you @Andrew565 with the prop detail above. I updated the PR description with some clarification too about how I tested it.